### ISSUE TITLE
Undefined symbol: _SDImageCoderEncodeCompressionQuality

### ISSUE THREAD

**Issue by Klingefjord** [OpenFlutter/flutter_image_compress#169]
This error gets thrown after trying to build an app that depends on `flutter_image_compress` from XCode. This is not deterministic, sometimes it goes away. Thinking it could me a mismatch of deployment targets?

```Undefined symbols for architecture x86_64:
  "_SDImageCoderEncodeCompressionQuality", referenced from:
      +[CompressHandler compressDataWithImage:quality:format:] in CompressHandler.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)```

**Comment by liuyasss** [comment]
Xcode Version 12.4 (12D4e)

Flutter 1.22.5 • channel stable • https://gitee.com/mirrors/Flutter.git
Framework • revision 7891006299 (7 weeks ago) • 2020-12-10 11:54:40 -0800
Engine • revision ae90085a84
Tools • Dart 2.10.4

I have the same issue!!!

**Comment by HTMHell** [comment]
https://github.com/OpenFlutter/flutter_image_compress/issues/160

**Comment by brazc** [comment]
Please try setting Runner's deployment target greater than 9.0

**Comment by themadmrj** [comment]
I had the same error a bunch of times. Making sure the settings (like bitcode, deployment version) were the same on my runner + extensions, then clean + rebuild, did the trick.

**Comment by AlexV525** [comment]
Should be fixed by #182 .
